tasks/components/OrganizationMemberList.tsx

```diff
@@ -220,7 +220,7 @@ export const OrganizationMemberList = ({
                   ),
                   name: (
                     <div className={tw('flex flex-row items-center h-12')}>
-                      <Avatar avatarUrl={cell.row.original.avatarURL} alt={cell.row.original.name} size="small"/>
+                      <Avatar avatarUrl={cell.row.original.avatarURL} alt={cell.row.original.name.charAt(0)} size="small"/>
```

Comment:
```suggestion
                      <Avatar avatarUrl={cell.row.original.avatarURL} alt={cell.row.original.name} size="small"/>
```
`.charAt(0)` of the original name does not seams like a proper alternate title.